<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLPage</class>
 <widget class="QDialog" name="DLPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="modelStackPage" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>8</number>
         </property>
         <item row="10" column="1">
          <widget class="QPlainTextEdit" name="logPlanText">
           <property name="lineWrapMode">
            <enum>QPlainTextEdit::WidgetWidth</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximumBlockCount">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="initBtn">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>初始化</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>操作</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>日志</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>模型</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="chooseModelBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>请选择模型文件</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>标签</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="labelsCombo">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>请输入标签名称</string>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QPushButton" name="startBtn">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QToolButton" name="clearInit">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/images/clear-l.png</normaloff>:/images/images/clear-l.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="referModeCombo"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>推理方式</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="modelTypesCombo"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>置信度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>算法</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>检测</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="chooseSourceBtn">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>请选择待检测资源</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="confSpinBox">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.250000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QToolButton" name="clearLabelsBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/images/clear-l.png</normaloff>:/images/images/clear-l.png</iconset>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>资源尺寸</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="resourceSizeSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>640</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QWGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
